sqlpage/templates/shell.handlebars

```diff
@@ -10,17 +10,56 @@
         <link rel="manifest" href="{{manifest}}">
     {{/if}}
     <link rel="stylesheet" href="{{static_path 'sqlpage.css'}}">
-   {{#each (to_array css)}}
+    {{#each (to_array css)}}
         {{#if this}}
             <link rel="stylesheet" href="{{this}}">
         {{/if}}
-   {{/each}}
+    {{/each}}
+
+    {{!-- Ensure font is an array --}}
+    {{#if font}}
+        {{#if (not (isArray font))}}
+            {{!-- Wrap single font string in an array --}}
+            {{#set "font" (array font)}}
+        {{/if}}
+    {{/if}}
+
+    {{#if font}}
+        <style>
+            {{#each font}}
+                {{#if (startsWith this "/")}}
+                    @font-face {
+                        font-family: '{{this}}_custom';
+                        src: url('{{this}}.woff2') format('woff2'),
```

Comment:
I think the path should already contain the extension. And we can say that we support only woff2.